chains/astar/server/Cargo.toml

```diff
@@ -14,7 +14,7 @@ futures = { version = "0.3", default-features = false, features = ["std"] }
 hex = "0.4"
 log = "0.4"
 parity-scale-codec = { workspace = true, features = ["derive"] }
-rosetta-config-astar.workspace = true
+rosetta-config-astar = { workspace = true, features = ["astar-metadata"] }
```

Comment:
The CI didn't catch this error because when testing using `--workspace` makes all crates use the same feature flags, I've updated the workflow to detect this kind of error in the future.